doc/metadata_routing.rst

```diff
@@ -17,16 +17,32 @@ Metadata Routing
     >>> import sklearn
     >>> sklearn.set_config(enable_metadata_routing=True)
 
-This guide demonstrates how metadata such as ``sample_weight`` can be routed
-and passed along to estimators, scorers, and CV splitters through
-meta-estimators such as :class:`~pipeline.Pipeline` and
-:class:`~model_selection.GridSearchCV`. In order to pass metadata to a method
-such as ``fit`` or ``score``, the object consuming the metadata, must *request*
-it. For estimators and splitters, this is done via ``set_*_request`` methods,
-e.g. ``set_fit_request(...)``, and for scorers this is done via the
-``set_score_request`` method. For grouped splitters such as
-:class:`~model_selection.GroupKFold`, a ``groups`` parameter is requested by
-default. This is best demonstrated by the following examples.
+This guide demonstrates how :term:`metadata`, such as ``sample_weight`` can be routed
+and passed between objects in scikit-learn. Metadata can be transfered to estimators,
+scorers, and CV splitters using :term:`meta-estimators` such as
+:class:`~pipeline.Pipeline` or :class:`~model_selection.GridSearchCV`.  It can also be
+routed between these objects without the presence of any meta-estimator. In order to
```

Comment:
what do you mean by 

> It can also be
routed between these objects without the presence of any meta-estimator.

Reply:
I added an explicit mention of functions that are able to route.